docs/faq.md

```diff
@@ -77,3 +77,7 @@ If you want to use Northstar you'll have to purchase Titanfall 2 either from Ste
 ### Q: Where is Frontier Defense?
 
 A: Frontier Defense is currently in development and will be available in a release when it's finished.
+
+### Q: I get an error message saying something along the lines of "SSL connect error" 
+
+A: There is currently no solution to this sadly, and we aren't sure what causes it. If you get this error, please create a ticket about it in the Northstar Discord server.
```

Comment:
I'd rephrase it to something along the lines of "we are actively investigating the error but have not found a solution yet. Please reach out to us via a ticket in the Northstar Discord so that we can gather more information on the issue"